### PR TITLE
Bubble Sort Ruby Program using yield

### PR DESCRIPTION
**Pull request by Abhigyan001**
- Created a bubble sort program in ruby. The first method is created using "while" loop and the elements are iterated inside it to sort the elements in increasing order.

- After the loop is over, the array is sorted and can be viewed using print or puts.

- The second method uses the same logic, For comparison, we used "yield" and ".positive?"  to determine the appropriate course of action.

- The array accepts strings as well as numbers as arguments and implements bubble sort to sort them.

### CODE REVIEW

File: bubblesort.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      my_arr[i + 1] = temp
+      condition = true
+    end
+  end
+  puts my_arr
+end
+
+bubble_sort_by(%w[bla ba blabla b blaaablaa]) do |left, right|
+  left.length - right.length
+end
```
Comment (kobiyoyo) [inline bubblesort.rb]: Let's remove this debug code.

File: bubblesort.rb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    (0...arr.length - 1).each do |i|
+      next unless yield(my_arr[i], my_arr[i + 1]).positive?
+
+      temp = my_arr[i]
+      my_arr[i] = my_arr[i + 1]
+      my_arr[i + 1] = temp
+      condition = true
+    end
+  end
+  puts my_arr
```
Comment (kobiyoyo) [inline bubblesort.rb]: Ruby methods automatically returns the last evaluated statement therefore, there is no need for `puts`. Let's remove it

**Review by kobiyoyo — CHANGES_REQUESTED** [review]
## Hi Team    
#### Great Job so far, everything works fine, but there are a few more things required. 
### General
- [ ]  Let's use a more concise title for this project in your read.MD file, for example  `Bubble Sort`.
- [ ] You should use an imperative form for commits instead of  past tense(For feature projects)
- [ ] You should use Github flow not Gitflow  ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/advanced_building_blocks_bubble_sort/git.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/57421588/))
- [ ] Check comments below
### Make changes and submit another code review request.
*Happy coding* 
(Any questions Feel free to contact me on slack @kobiyoyo)

**Review by voscarmv — APPROVED** [review]
# Status: Project Approved ✨ 😎👍 💯 ✔️
### You may merge this branch and move forward ⛙🛣

### Submit these changes and request a new code review! 😄 👍 

### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻 
# Happy Hacking! 🧠 ⌨️ 🖱️ 🖥️

**Pull request merged**